config/addon/addon-config.yaml

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
```suggestion
# Copyright 2020 The KubeCarrier Authors.
```

Reply:
We use this `KubeCarrier Authors` in this project: https://github.com/kubermatic/kubecarrier/blob/master/LICENSE#L189-L201